tests/testsuite/workspaces.rs

```diff
@@ -1251,7 +1230,14 @@ fn rebuild_please() {
     p.cargo("run")
         .cwd("bin")
         .with_status(101)
-        .with_stderr_contains("[..]assertion[..]")
+        .with_stderr_data(str![[r#"
+...
+assertion `left == right` failed
+  left: 1
+ right: 0
+[NOTE] run with `RUST_BACKTRACE=1` environment variable to display a backtrace
+...
+"#]])
```

Comment:
We try to minimize rustc/std output that we match because we support multiple versions of rustc and changing rustc's behavior can be more difficult when cargo tests are dependent on it.  We should probably match on as little as possible.

For some cases, we don't even think about this.  I'm only thinking about this one because the output recently changed.

Reply:
Ok, I'll update to use the original matching pattern.